managed-configuration-chrome.py

```diff
@@ -0,0 +1,118 @@
+# Sample code to use managed configuration APIs of Esper.
+# This code run the APIs on the chrome browser and help in
+# 1. Place specific URLs on an allowlist
+# 2. Place specific URLs on a blocklist
+# 3. Disable browser incognito mode on Android
+# 4. Enable forced Safe Search on a device
+
+# Code can be run either on set of devices or set of groups
+# provided as a argument to the command request
+
+
+import esperclient
+from esperclient.rest import ApiException
+from esperclient.models.v0_command_args import V0CommandArgs as CommandArgs
+
+# Configure API key authorization: apiKey
+configuration = esperclient.Configuration()
+configuration.host = 'https://<endpoint-name>-api.esper.cloud/api'
+configuration.api_key['Authorization'] = '<API-Key>'
+configuration.api_key_prefix['Authorization'] = 'Bearer'
+
+# create an instance of the API class
+api_instance = esperclient.CommandsV2Api(esperclient.ApiClient(configuration))
+enterprise_id = '<Enterprise ID>' # str | ID of the enterprise
+
+# Modify this Python Dict to set your own managed configurations
+command_args = CommandArgs(
+    custom_settings_config={
+        "managedAppConfigurations": {
+            "com.android.chrome": {
+                "URLBlacklist": ["https://www.omegle.com/","Facebook.com","instagram.com"], # not allow these websites to work
+                "URLWhitelist": ["*"], # rest all allowed to open
+                "IncognitoModeAvailability": "1",   # disable incognito mode
+                "ForceGoogleSafeSearch" : "true",   # enable safe search to work
+                "HomepageLocation": "https://esper.io"  # set chrome home page
+            }
+        }
+    }
+)
+
+# int | Number of results to return per page. (optional) (default to 20)
+configuration.per_page_limit = 20
+# int | The initial index from which to return the results.(optional) default to 0)
+configuration.per_page_offset = 0
+
+
+
+# get all the groups present in the enterprise and execute run_managed_configuration on each of them
+def get_all_groups_in_enterprise():
+    # create an instance of the API class
+    allgroups = []
+    api_instance = esperclient.DeviceGroupApi(esperclient.ApiClient(configuration))
+    try:
+        api_response = api_instance.get_all_groups(enterprise_id,
+                                                    limit=configuration.per_page_limit,
+                                                    offset=configuration.per_page_offset)
+        if len(api_response.results):
+            for group in api_response.results:
+                # add all the devices in this group to global list of devices
+                allgroups.append(group.id)
+                run_managed_configuration(allgroups)
```

Comment:
Here the value of 'allgroups' is like this, [1], [1,2], [1,2,3]...etc.
You have to use singleton list to 'run_managed_configuration' function if you're looking to create command request for each group.
run_managed_configuration([group.id])

Reply:
done.